packages/taiko-client/prover/proof_submitter/proof_submitter.go

```diff
@@ -477,7 +477,12 @@ func (s *ProofSubmitter) AggregateProofs(ctx context.Context) error {
 				startTime,
 			)
 			if err != nil {
-				log.Error("Failed to request proof aggregation", "err", err)
+				if errors.Is(err, proofProducer.ErrProofInProgress) ||
+					errors.Is(err, proofProducer.ErrRetry) {
+					log.Info("Aggregating proofs", "status", err)
```

Comment:
Shall we also log the length of the batch and the proof type here?

Reply:
OK